pinact: 1.0.0 -> 1.1.2 and set CGO_ENABLED=0

kachick opened the pull request:
Diff: https://github.com/suzuki-shunsuke/pinact/compare/v1.0.0...v1.1.2

<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [x] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

kachick committed:
pinact: prefer versionCheckHook rather than testers.testVersion

https://github.com/NixOS/nixpkgs/blob/bd352cd67834c983d69adf4080d225dbe28f99ba/doc/stdenv/passthru.chapter.md?plain=1#L84

kachick committed:
pinact: prefer tag rather than rev for fetchFromGitHub

introduced in 7822cb34f86bcdd0b9f31f2955062216857a6757

kachick committed:
pinact: set CGO_ENABLED=0

Upstream sets 0 in the release

https://github.com/suzuki-shunsuke/pinact/blob/v1.0.0/.goreleaser.yml#L13

nixpkgs sets 1 by default

https://github.com/NixOS/nixpkgs/blob/471addceb995f4f9e84618c471d07c97597da2c5/doc/languages-frameworks/go.section.md?plain=1#L252

kachick committed:
pinact: reduce scope of with in meta

e671d3bbbda9b2bec8fa119aa1ff46499118ef17

kachick commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 368078`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>pinact</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>pinact</li>
  </ul>
</details>

kachick committed:
pinact: 1.0.0 -> 1.1.0

Diff: https://github.com/suzuki-shunsuke/pinact/compare/v1.0.0...v1.1.0

kachick commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 368078`


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>pinact</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>pinact</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>pinact</li>
  </ul>
</details>

kachick committed:
pinact: 1.1.0 -> 1.1.1

Diff: https://github.com/suzuki-shunsuke/pinact/compare/v1.1.0...v1.1.1

kachick commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 368078`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>pinact</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>pinact</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>pinact</li>
  </ul>
</details>

kachick committed:
pinact: 1.1.1 -> 1.1.2

Diff: https://github.com/suzuki-shunsuke/pinact/compare/v1.1.1...v1.1.2

kachick commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 368078`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>pinact</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>pinact</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>pinact</li>
  </ul>
</details>

katexochen commented:
I think the `gen-jsonschema` binary shouldn't be part of the output, right? Maybe set [`subPackages = [  "cmd/pinact" ];`](https://nixos.org/manual/nixpkgs/unstable/#var-go-subPackages).

```
❯ l ./results/pinact-x86_64-linux/bin/
Permissions Size User Date Modified Name
.r-xr-xr-x  3.7M root  1 Jan  1970  gen-jsonschema
.r-xr-xr-x   11M root  1 Jan  1970  pinact
```

kachick committed:
pinact: exclude bin/gen-jsonschema

Which was introduced in version 1.1.2
However it looks like a project internal tool

For example, homebrew did not include it
https://github.com/Homebrew/homebrew-core/blob/2e88df3e7a7a39f4d0cd462a0f46d73861d7af8d/Formula/p/pinact.rb#L20-L28

kachick commented:
Thank you! :pray:
